{"data_version": "4.0","references": {"reference_data": [{"url": "https://www.imperva.com/blog/scapy-sploit-python-network-tool-is-vulnerable-to-denial-of-service-dos-attack-cve-pending/"},{"url": "https://github.com/secdev/scapy/pull/1409/files#diff-441eff981e466959968111fc6314fe93L1058"}]},"description": {"description_data": [{"lang": "eng","value": "scapy version 2.4.0 and earlier contains a Denial of Services vulnerability in  _RADIUSAttrPacketListField class that can result in busy loop forever . This attack appear to be exploitable via a packet sent over the network or in a pcap. This vulnerability appears to have been fixed in after commit 0d7ae2b039f650a40e511d09eb961c782da025d9."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "2.4.0 and earlier"}]},"product_name": "scapy"}]},"vendor_name": "scapy"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2019-02-18T19:06:00.120147","DATE_REQUESTED": "2019-02-03T18:35:13","ID": "CVE-2019-1000037","ASSIGNER": "kurt@seifried.org","REQUESTER": "koby.kilimnik@imperva.com"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "Denial of Services"}]}]}}